src/build/iconGenerator.js

```diff
@@ -113,9 +163,30 @@ function buildJsonStructure(iconsFolderBasePath) {
  */
 function getDefaultSchema(iconsFolderBasePath) {
   /* eslint-disable no-underscore-dangle */
-  defaultSchema.iconDefinitions._file.iconPath = iconsFolderBasePath + 'file.svg';
-  defaultSchema.iconDefinitions._folder.iconPath = iconsFolderBasePath + 'folder.svg';
-  defaultSchema.iconDefinitions._folder_open.iconPath = iconsFolderBasePath + 'folder_opened.svg';
+
+  // dark theme
+  defaultSchema.iconDefinitions._file
+    .iconPath = iconsFolderBasePath + 'file.svg';
+  defaultSchema.iconDefinitions._folder
+    .iconPath = iconsFolderBasePath + 'folder.svg';
+  defaultSchema.iconDefinitions._folder_open
+    .iconPath = iconsFolderBasePath + 'folder_opened.svg';
+
+  // light theme
+  // default file and folder related icon paths if not set,
```

Comment:
For more references see:
http://www.hongkiat.com/blog/source-code-comment-styling-tips/
http://javascript.crockford.com/code.html